requirements.txt

```diff
@@ -1,5 +1,5 @@
-PyJWT==1.1.0
-slumber==0.7.1
+# Package requirements
+-e .
```

Comment:
we didn't install them as editable before and I don't think it's desired to have them install that way in production.  locally, one can modify or manually uninstall and reinstall as editable.